test/Test/Data/Generics/Rep.purs

```diff
@@ -0,0 +1,187 @@
+module Test.Data.Generics.Rep where
```

Comment:
The package was named generics-rep but the module is Data.Generic.Rep 😬 Should we rename this module to Test.Data.Generic.Rep instead (and testGenericsRep to testGenericRep)? Perhaps you wanted to emphasize that we’re testing multiple generic implementations though, in which case that’s fine by me.

Reply:
Ah, darn it! Nope, I just spelled it wrong because I was going by the repo name.